packages/forms-web-app/src/controllers/common/enter-code.js

```diff
@@ -173,8 +182,14 @@ const postEnterCode = (views) => {
 			});
 		}
 
+		const isReturningUser = req.session.appeal.newOrSavedAppeal === 'return';
```

Comment:
could we create a separate file (appeal-user-service.js or something along those lines) that handles appeal user based session values, being able to see in one place would help understanding in future of what can be set on a session

Reply:
Good idea, will add that. So something like:

exports.setNewOrSavedAppeal = (session, value) => {
 session.appeal.newOrSavedAppeal === value;
};